### PR TITLE
update param types in artsearch

### PR DESCRIPTION
**Pull request by mateeullahmalik**

### CODE REVIEW

**Review by danileng20 — APPROVED** [review]
LGTM!

File: walletnode/api/design/artworks.go [hunk 1]
```diff
@@ -503,26 +505,39 @@ var SearchArtworkParams = func() {
 		Maximum(1000)
 		Example(1000)
 	})
-	Attribute("min_nsfw_score", Int, func() {
```
Comment (pjenys0) [inline walletnode/api/design/artworks.go:506]: nit: can change to nsfw_score to open_nsfw_score to easy to track?

**Review by pjenys0 — APPROVED** [review]
LGTM.
but did you test with actual cnode?

**Review by danileng20 — APPROVED** [review]

**Review by leonidcode20 — APPROVED** [review]
LGTM!

**Pull request merged**